pkg/cmd/describe/describe_rollout.go

```diff
@@ -472,8 +514,128 @@ func (o *DescribeRolloutOptions) printTrafficRouting(trafficRouting []rolloutsap
 	}
 }
 
-func (o *DescribeRolloutOptions) printRolloutInfo(rollout *rolloutsapi.Rollout) {
-	// Name, Namespace, Status, Message
+func (o *DescribeRolloutOptions) printRolloutV1AlphaInfo(rollout *rolloutsapiv1alpha1.Rollout) {
+	fmt.Fprintf(o.Out, tableFormat, "Name:", rollout.Name)
+	fmt.Fprintf(o.Out, tableFormat, "Namespace:", rollout.Namespace)
+	if rollout.Status.ObservedGeneration == rollout.GetObjectMeta().GetGeneration() {
+		fmt.Fprintf(o.Out, tableFormat, "Status:", o.colorizeIcon(string(rollout.Status.Phase))+" "+string(rollout.Status.Phase))
+
+		if rollout.Status.Message != "" {
+			fmt.Fprintf(o.Out, tableFormat, "Message:", rollout.Status.Message)
+		}
+	}
+
+	// Strategy
+	fmt.Fprintf(o.Out, tableFormat, "Strategy:", "Canary")
+
+	if canary := rollout.Spec.Strategy.Canary; canary != nil {
+		fmt.Fprintf(o.Out, tableFormat, " Step:", strconv.Itoa(int(rollout.Status.CanaryStatus.CurrentStepIndex))+"/"+strconv.Itoa(len(canary.Steps)))
+		fmt.Fprint(o.Out, " Steps:\n")
+		currentStepIndex := int(rollout.Status.CanaryStatus.CurrentStepIndex)
+		for i, step := range canary.Steps {
+			isCurrentStep := (i + 1) == currentStepIndex
+			if isCurrentStep {
+				fmt.Fprint(o.Out, "\033[32m")
+			}
+
+			if step.Replicas != nil {
+				fmt.Fprintf(o.Out, tableFormat, "  -  Replicas: ", step.Replicas)
+			}
+			if step.TrafficRoutingStrategy.Weight != nil {
+				fmt.Fprintf(o.Out, tableFormat, "     Traffic: ", *step.TrafficRoutingStrategy.Weight)
+			}
+
+			if step.Matches != nil {
+				fmt.Fprintln(o.Out, "     Matches: ")
+				for _, match := range step.Matches {
+					fmt.Fprintln(o.Out, "      - Headers: ")
+					for _, path := range match.Headers {
+						fmt.Fprintf(o.Out, tableFormat, "       - Name:", path.Name)
+						fmt.Fprintf(o.Out, tableFormat, "         Value:", path.Value)
+						fmt.Fprintf(o.Out, tableFormat, "         Type:", *path.Type)
+					}
+				}
+			}
+
+			if isCurrentStep {
+				fmt.Fprintf(o.Out, tableFormat, "     State:", rollout.Status.CanaryStatus.CurrentStepState)
+				fmt.Fprint(o.Out, "\033[0m") // Reset color
+			}
+		}
+	}
+
```

Comment:
plz also print the trafficroutings info:
1. spec.strategy.canary.trafficRoutings
2. annotation rollouts.kruise.io/trafficrouting, which is equivalent as rollout.Spec.Strategy.Canary.TrafficRoutingRef in v1beta1

Reply:
Fixed v1alpha1 and ensured uniformity.